File: health-api.py
#!/usr/bin/python3

import requests
category = 'health'
api_url = 'https://api.api-ninjas.com/v1/quotes?category={}'.format(category)
response = requests.get(api_url, headers={'X-Api-Key': 'Jnfv5j/HghjhDtRkbTzY3A==hGVTan9WvBtclTMQ'})
if response.status_code == requests.codes.ok:
    print(response.text)
else:
    print("Error:", response.status_code, response.text)

File: health-api.js
#!/usr/bin/env node

// Defines the category of the quotes
const category = 'Health';
// API key for the authentication
const apiKey = 'Jnfv5j/HghjhDtRkbTzY3A==hGVTan9WvBtclTMQ';
// URL for the API request to fetch the quotes
const apiUrl = `https://api.api-ninjas.com/v1/quotes?category=${category}`;

// Function to  fetch a new quote from the API
async function fetchQuote() {
    try {
        // Make a request to the API
        const response = await fetch(apiUrl, {
            headers: {
                'X-Api-Key': apiKey // Includes the API key in the requests
            }
        });
        // Check if the responses is ok (status code 200-209)
        if (!response.ok) {
            throw new Error(`HTTP error! status: ${response.status}`);
        }
        // Parse the JSON data from the response
        const data = await response.json();
        // Get the first quote from the data
        const quote = data[0];
        // Update the quote text and the author on the page
        document.getElementById('quote-text').textContent = quote.quote;
        document.getElementById('quote-author').textContent = `— ${quote.author}`;
    } catch (error) {
        // Handle errors
        console.error('Error fetching quote:', error);
        // Displays error messages on the page 
        document.getElementById('quote-text').textContent = 'Error fetching quote';
        document.getElementById('quote-author').textContent = '';
    }
}
// Fetch a quote when the page loadss
fetchQuote();